lib/autorest/operations/datasets.js

```diff
@@ -1515,14 +1520,14 @@ Datasets.prototype.setAllConnections = function (collectionName, workspaceId, da
               required: false,
               serializedName: 'ObjectElementType',
               type: {
-                name: 'String'
+                name: 'Object'
```

Comment:
I think that's just an output change due to using AutoRest 0.16.0 instead of AutoRest 0.15.0